File: data/src/main/java/com/thinkzi/weather/data/model/Snow.java
package com.thinkzi.weather.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Snow {

    @SerializedName("snowVolume")
    @Expose
    private double snowVolume;

    public Snow() {
        this.snowVolume = 0.0;
    }

    public double getSnowVolume() {
        return snowVolume;
    }

    public void setSnowVolume(double _snowVolume) {
        this.snowVolume = snowVolume;
    }
}

File: data/src/main/java/com/thinkzi/weather/data/model/Snow.kt
package com.thinkzi.weather.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Snow {

    @SerializedName("snowVolume")
    @Expose
    internal var snowVolume: Double = 0.toDouble()

    init {
        this.snowVolume = 0.0
    }

    fun getSnowVolume(): Double {
        return snowVolume
    }

    fun setSnowVolume(_snowVolume: Double) {
        this.snowVolume = _snowVolume
    }
}
